English translation (#71)
personium/new-docs: src/app-developer/README.md

---
id: README
title: Guide for App Developers

> sidebar_label: A guide for app developers

Reviewer: ```suggestion
sidebar_label: Guide for app developers
```

---

> A document for people who develop apps using Personium's API.

Reviewer: ```suggestion
This document is for developers who develop apps using Personium's APIs.
```

## Introduction to app development

To learn the basics of app development, refer to the following documents in Getting Started.

1. [Introduction](../getting-stared/appdev-introduction.md)
2. [Unit Manager](../getting-stared/appdev-management-tool.md)
3. [App development flow](../getting-stared/appdev-process.md)
4. [App implementation](../getting-stared/appdev-impl.md)

## App development details

> Details of app development are divided by topic.

Reviewer: ```suggestion
Details of app development are divided in the following categories.
```

- Authorization
  - [Authorization model](../user_guide/003_Auth.md)
  - [Role in Personium](./Roles.md)
  - [Access control by RBAC](../apiref/006_Access_Control.md)
  - [App Authentication](./app_authn.md)
  - [Single Sign On](./launch_from_homeapp.md)
- Box/bar file
  - [Box installation](../apiref/007_Box_install.md)
  - [bar file](../apiref/301_Bar_File.md)
- Collection
  - [WebDAV](../user_guide/007_WebDAV_model.md)
  - [OData](./using_odata.md)
  - [Personium Engine](./Personium-Engine.md)
  - [Engine Script samples](./671_Engine_Script_Samples.md)
- [Cell GUI Developer's Guide](../cell-gui-developer/README.md)

## Sample app

> Some sample applications have been released as JavaScript applications for web browsers.

Reviewer: ```suggestion
Some sample apps have been released as JavaScript web apps.
```
